lib/components/viewers/live-stop-times.tsx

```diff
@@ -144,36 +169,72 @@ class LiveStopTimes extends Component<Props, State> {
       return stopTimeComparator(stopTimesA[0], stopTimesB[0])
     }
 
+    const routeTimes = Object.values(stopTimesByPattern)
+      .filter(({ times }) => times.length !== 0)
+      .sort(patternComparator)
+      .map((route) => {
+        const sortedTimes = route.times
+          .concat()
+          ?.sort(stopTimeComparator)
+          // Only show times within 24 hours of next arrival time
+          .filter((time: any, i: number, array: Array<any>) => {
+            const firstDepartureTime =
+              array[0].serviceDay + array[0].realtimeDeparture
```

Comment:
```suggestion
              array?.[0]?.serviceDay + array?.[0]?.realtimeDeparture
```

We can't rely on any of this data!